GameServerLib/GameObjects/Spells/Spell.cs

```diff
@@ -67,6 +65,11 @@ public Spell(Game game, Champion owner, string spellName, byte slot)
             _spellGameScript.OnActivate(owner);
         }
 
+        void ISpell.Deactivate()
```

Comment:
I personally prefer having access level defined (`private` / `public`) explicitly, but I'm not sure if we have a guideline on that 🤔